writer.js

```diff
@@ -25,6 +25,7 @@ Writer.prototype.buildStatusMessage = function buildStatusMessage(statusInfo) {
   if (statusInfo.error) {
     statusInfo.eventType = 'error';
     statusInfo.message = statusInfo.error.message;
+    statusInfo.output = statusInfo.error.output;
```

Comment:
I feel we should either append this to the `message` itself or have a separate property which will act as the longer form text for any of these events. I think this is probably the one event that if it errors it will have a large stack to deal with. Thoughts @msluther @fritzmonkey? Its relevant when thinking about the future UI.

Reply:
I could see either. I would tend towards having a separate field for something like a "message details" sort of thing and letting the UI decide how that renders.

Reply:
@fritzmonkey ok, lets rename this to `details` and that will be another top level property in the model. cc @msluther